tests/dualtor/test_switchover_failure.py

```diff
@@ -137,6 +137,11 @@ def common_setup_teardown(
 
     rand_selected_dut.shell_cmds(cmds=cmds)
 
+    # If the test was skipped then early exit from teardown
+    if "rep_call" in request.node.__dict__ and request.node.rep_call.skipped or \
```

Comment:
pls use `hasattr` to check the attribute existence, as:
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/dualtor/test_switchover_failure.py#L140-L146

Reply:
Done